x-pack/solutions/security/plugins/security_solution/public/timelines/components/timeline/tabs/pinned/index.tsx

```diff
@@ -78,10 +83,22 @@ export const PinnedTabContentComponent: React.FC<Props> = ({
   const [pageIndex, setPageIndex] = useState(0);
 
   const { telemetry } = useKibana().services;
-  const { dataViewId, sourcererDataView, selectedPatterns } = useSourcererDataView(
+
+  const { newDataViewPickerEnabled } = useEnableExperimental();
+
+  let { dataViewId, sourcererDataView, selectedPatterns } = useSourcererDataView(
     SourcererScopeName.timeline
   );
 
+  const experimentalSelectedPatterns = useSelectedPatterns(SourcererScopeName.timeline);
+  const { dataView: experimentalDataView } = useDataView(SourcererScopeName.timeline);
+
+  if (newDataViewPickerEnabled) {
+    selectedPatterns = experimentalSelectedPatterns;
+    sourcererDataView = experimentalDataView;
+    dataViewId = experimentalDataView.id ?? '';
```

Comment:
same thing here re defaulting id to `''` in the `experimentalDataView` initialization